README.md

```diff
@@ -6,7 +6,7 @@
 >
 > Alternatively, watch a [talk and demo](http://begriffs.com/posts/2016-01-20-ihaskell-notebook.html) showing off IHaskell features.
 
-IHaskell is a kernel for the [Jupyter project](http://ipython.org), which allows you to use Haskell inside Jupyter frontends (including the console and notebook). It currently supports GHC 8.
+IHaskell is a kernel for the [Jupyter project](http://ipython.org), which allows you to use Haskell inside Jupyter frontends (including the console and notebook). It currently supports GHC 8. For GHC 7.10 support please use the `GHC7` tag.
```

Comment:
I would also link to [the Github page for the tag](https://github.com/gibiansky/IHaskell/releases/tag/GHC7) at this point. Not everybody interested in installing IHaskell may even know what a Git tag is, with a link they can at least download a tarball.

Reply:
Good point, updated.